apps/condo/domains/organization/utils/name.utils.js

```diff
@@ -0,0 +1,8 @@
+const SPECIAL_CHARS_REGEXP = /[`!@#$%^&*()_+\-=[\]{};':"\\|,.<>/?~]/
+const NUMBERS_REGEXP = /[0123456789]/
+
+const isValidName = (name) => ![...(name.trim())].every(char => SPECIAL_CHARS_REGEXP.test(char) || NUMBERS_REGEXP.test(char))
```

Comment:
looks like not optimal variant of using regex

If you want to test name for special chars
```
const test = name.match(/\W|_/g)
```
If not contains = result will be null
if contains = result will be array of symbols

Reply:
Also if you don't need match object to work later with, then test is ok, since it return true/false, which is more obvious to read.

Reply:
Why the dot (.) and comma (,) or quotes (“‘"’') is not allowed? Look at real organization names.

Reply:
> Why the dot (.) and comma (,) or quotes (“‘"’') is not allowed? Look at real organization names.

Only names that consist only of special characters or numbers are prohibited. Of course, for example, the name LLC "Example" is allowed

Reply:
fixed